### PR TITLE
Re-organises food

### PR DESCRIPTION
**Pull request by MarinaGryphon**
## Description of changes

- Repaths `/obj/item/chems/food/drinks` to `/obj/item/chems/drinks`
- Repaths `/obj/item/chems/food/condiment` to `/obj/item/chems/condiment`
- Repaths `/obj/item/chems/food/snacks` to `/obj/item/chems/food`

This technically does include functional changes, but honestly they're basically bugfixes: you can no longer add reagents to non-`OPEN_CONTAINER` beverage cans with a syrigne or dropper. You *can* add reagents to `OPEN_CONTAINER` cans/condiment containers, and to all `/food` subtypes.
## Why and what will this PR improve

Makes it easier to find things, removes a useless semantic type.

## Authorship

- MoondancerPony/jade2562/koboldlove

### CODE REVIEW

File: maps/nexus/nexus-2.dmm [hunk 1]
```diff
@@ -123,7 +123,7 @@
 "Nq" = (/obj/machinery/atmospherics/pipe/simple/hidden/supply,/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,/obj/machinery/door/airlock/hatch/autoname/general,/obj/machinery/door/firedoor,/turf/simulated/floor/tiled/freezer,/area/nexus/civilian/hygiene)
 "NF" = (/obj/effect/floor_decal/corner/b_green/diagonal,/obj/machinery/light{dir = 1},/turf/simulated/floor/tiled,/area/nexus/civilian/bar)
 "NM" = (/obj/structure/catwalk,/obj/machinery/computer/cryopod{pixel_y = -32},/turf/simulated/floor/plating,/area/nexus/civilian/hygiene)
-"NT" = (/obj/structure/table/standard,/obj/item/chems/food/condiment/small/packet/salt,/obj/item/chems/food/condiment/small/packet/pepper,/obj/random/drinkingglass,/obj/effect/decal/cleanable/dirt,/turf/simulated/floor/glass,/area/nexus/civilian/bar)
+"NT" = (/obj/structure/table/standard,/obj/item/chems/condiment/small/packet/salt,/obj/item/chems/condiment/small/packet/pepper,/obj/random/drinkingglass,/obj/effect/decal/cleanable/dirt,/turf/simulated/floor/glass,/area/nexus/civilian/bar)
```
Comment (afterthought2) [inline maps/nexus/nexus-2.dmm]: Note to self: why is this map in this format?

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -1,5 +1,12 @@
-//Food items that are eaten normally and don't leave anything behind.
+////////////
+// Snacks //
```
Comment (afterthought2) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Aren't snacks anymore.

File: code/modules/reagents/reagent_containers/food/snacks/dough.dm [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+	bitesize = 2
+	center_of_mass = @"{'x':16,'y':13}"
+	nutriment_desc = list("dough" = 3)
+	nutriment_amt = 3
+	nutriment_type = /decl/material/liquid/nutriment/bread
+
+// Dough + rolling pin = flat dough
+/obj/item/chems/food/snacks/dough/attackby(obj/item/W, mob/user)
+	if(istype(W,/obj/item/kitchen/rollingpin))
+		new /obj/item/chems/food/snacks/sliceable/flatdough(src)
```
Comment (afterthought2) [inline code/modules/reagents/reagent_containers/food/snacks/dough.dm]: Should maybe make all of these spawn in `get_turf`, or use `dropInto`. It does work due to a `/snacks/Destroy` override, but that override is also kind of undesirable. `loc` also doesn't really work, as that's probably the mob.

File: code/modules/reagents/reagent_containers/food/snacks/baked_goods.dm [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+	name = "banana cream pie"
+	desc = "Just like back home, on clown planet! HONK!"
+	icon_state = "pie"
+	trash = /obj/item/trash/plate
+	filling_color = "#fbffb8"
+	center_of_mass = @"{'x':16,'y':13}"
+	nutriment_desc = list("pie" = 3, "cream" = 2)
+	nutriment_amt = 4
+	bitesize = 3
+/obj/item/chems/food/snacks/pie/Initialize()
```
Comment (afterthought2) [inline code/modules/reagents/reagent_containers/food/snacks/baked_goods.dm]: There are some missing newlines; not going to comment on every one.

**Review by afterthought2 — COMMENTED** [review]
Should remove `/snacks` from the file path as well, probably.

File: code/modules/reagents/reagent_containers/food/snacks/baked_goods.dm (hunk 4 above)
Comment (MarinaGryphon) [reply code/modules/reagents/reagent_containers/food/snacks/baked_goods.dm]: *relatively* sure that these existed in the already-existing similarly-grouped runs that I copied from `snacks.dm`, but I can go through and fix these if you'd like.

Comment (afterthought2) [reply code/modules/reagents/reagent_containers/food/snacks/baked_goods.dm]: Yeah, the old formatting is bad (see all the `.=..()` in Initialize) but having the whitespace be normal at least would be nice.

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**